### ISSUE TITLE
Any workaround for "Dont support type Padding" for OpenCL backend?

### ISSUE THREAD

**Issue by wangg** [alibaba/MNN#761]
# 平台(如果交叉编译请再附上交叉编译目标平台):  
RK3399, Debian

# Github版本:
0.2.2.3

# 编译方式:

```
cmake -DMNN_OPENMP=ON -DMNN_OPENCL=ON -DMNN_BUILD_BENCHMARK=true -DMNN_BUILD_QUANTOOLS=ON -DMNN_BUILD_CONVERTER=true -DMNN_BUILD_DEMO=ON -DMNN_SEP_BUILD=ON -DMNN_BUILD_SHARED_LIBS=ON -DMNN_USE_SYSTEM_LIB=ON ..
```

# Tensorflow版本
1.13.2

tf.keras.applications.MobileNetV2转换成MNN后，用MNNV2Basic.out测试，OpenCL要比CPU慢非常多，大量时间消耗在Pad上并提示"Don't support type Padding"。tensorflow的模型里用了pad补零，看起来是Pad最终在CPU上执行，来回交换数据消耗了时间？不知可有解决方法？

**Comment by Kay-Tian** [comment]
Vulkan版本也不支持pad操作，想请教一下是因为GPU实现有困难还是单纯的尚未实现？

**Comment by jxt1234** [comment]
已经支持了
